spring-batch-infrastructure/src/test/java/org/springframework/batch/item/file/transform/DelimitedLineTokenizerTests.java

```diff
@@ -47,6 +47,11 @@ public void testTokenizeRegularUse() {
 	}
 
 	@Test
+	public void testEmptyString() {
+		tokenizer.tokenize("   ");
```

Comment:
It reproduces the bug which is a `ArrayIndexOutOfBoundsException`.
I'm not familiar with your code style. Do you want me to wrap it in a try/catch and fail in the catch?